share/wake/lib/core/json.wake

```diff
@@ -155,10 +155,32 @@ export def x // y =
       foldr flatten tail l
   JArray (helper Nil x)
 
+# Compares two json values for equality. Returns
+# True if the two json values are identical and
+# false otherwise. In JavaScript this would be
+# called "deep equality".
+#
+# Examples:
+# ```
+#   JInteger 10 ==/ JInteger 10 = True
+#   JInteger 11 ==/ JInteger 10 = False
+#   JInteger 10 ==/ JString "foo" = False
+#   JDouble 10.0 ==/ JInteger 10 = False
```

Comment:
Comment is wrong.

Reply:
Oh yeah it was right and then I decided I didn't like that behavior so I fixed it. Thanks!